knowledge-base/dropdownlist-change-icon-when-popup-opened.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Changing the DropDownList Icon When the Popup is Opened
```

Comment:
Developers are more likely to use "change" instead of "changing" in a Google search. It's also shorter.

If you wish to include both keywords, then use "changing" inside the article.